src/main/scala/temple/generate/server/go/service/main/GoServiceMainStructGenerator.scala

```diff
@@ -23,27 +23,27 @@ object GoServiceMainStructGenerator {
   private def generateValidatorAnnotation(attrType: AttributeType): String =
     s"`valid:${doubleQuote(
       attrType match {
-        case AttributeType.ForeignKey(_) | AttributeType.UUIDType => "type(string),uuid,required"
-        case AttributeType.BoolType                               => "type(bool),required"
-        case AttributeType.DateType                               => "type(string),required"
-        case AttributeType.DateTimeType                           => "type(string),rfc3339,required"
-        case AttributeType.TimeType                               => "type(string),required"
-        case AttributeType.BlobType(_)                            => "type(string),base64,required"
+        case AttributeType.ForeignKey(_) | AttributeType.UUIDType => "type(*string),uuid,required"
```

Comment:
```suggestion
        case AttributeType.ForeignKey(_) | AttributeType.UUIDType => "type(*uuid.UUID),uuid,required"
```

Should this be this? Hard to know from docs

Reply:
I'm pretty sure it should